knowledge-base/live-traffic-pane.md

```diff
@@ -57,19 +54,30 @@ Use the **>_ Terminal** button to capture traffic from a specific terminal insta
 
 [Learn more about the independent terminal capturing...]({%slug capture-terminal-traffic%})
 
+### Clearing Sessions
+
+The **Clear** button will remove all captured traffic from the **Live Traffic** grid. The action requires explicit confirmation. To remove only specific sessions, select the targeted sessions and use the **Clear** option from the context menu.
 
-### Removing All Sessions
+You can also clear only a specific set of sessions based on some pre-defined criteria. The complete list of **Clear xxx** options is available through the dropdown menu next to the **Clear** button, and the available options are as follows:
 
-The **Remove All** button deletes all captured traffic from the **Live Traffic** grid. The action requires explicit confirmation. To remove only specific sessions, select the desired ones and use the remove options from the context menu.
+- **Clear All**&mdash;Removes all entries from the traffic grid (an analog to the **Clear** button).
+- **Clear Closed**&mdash;Removes all sessions with session state "Closed".
+- **Clear Visible**&mdash;Removes all sessions currently visible in the grid. Will preserve filtered sessions (these sessions will return if the filters are cleared).
+- **Clear Filtered Out**&mdash;Removes all sessions that are filtered.
+- **Clear CONNECTS**&mdash;Removes all sessions that use HTTP method CONNECT (also referred to as proxy tunnels).
+- **Clear XXX**&mdash;Removes all sessions where the received HTTP status code is XXX.
+- **Clear Static Resources**&mdash;Removes all sessions that return static resources like text or images.
+- **Clear Marked**&mdash;Removes all sessions from the grid marked through a rule or from the context menu.
```

Comment:
here, we have something specific that may cause confusion -> we assume that CONNECTs are marked (because they actually are), and clear them too. We may have to mention this so we don't cause frustration